schemas/context/implementation.schema.json

```diff
@@ -0,0 +1,35 @@
+{
+  "meta:license": [
+    "Copyright 2017 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id": "https://ns.adobe.com/xdm/context/implementation",
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "title": "Implementation",
+  "type": "object",
+  "meta:extensible": true,
+  "description": "Implementation of an API.",
+  "definitions": {
+    "implementation": {
+      "properties": {
+        "xdm:name": {
+          "title": "Name",
+          "type": "string",
+          "description": "SDK or endpoint Name, e.g sCode, AppMeasurement."
```

Comment:
It looks like we already know a couple of values that this property can take on. Documenting it with `meta:enum` will make this knowledge more readily accessible and reduce client implementation errors.